teams/core-observers.toml

```diff
@@ -0,0 +1,14 @@
+name = "coore-observers"
+subteam-of = "core"
+
+[people]
+leads = []
+members = [
+    "skade",
+]
+
+[permissions]
```

Comment:
Just a note, the core team has the `bors.rust.review` permission. Not sure if you want to put that here.

Reply:
I don't know either :)

Reply:
I think it's fine. I would expect that -- like anyone with r+ -- there is an expectation that people will only use it where appropriate. But I could see a core team observer being delegated some task by core team that involves r+.

Reply:
Added. Ready to merge, anyone want to push the button?